tests/kernel_bundle/sycl_is_compatible.cpp

```diff
@@ -0,0 +1,240 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+//  Provides tests for is_compatible()
+//
+*******************************************************************************/
+
+#include "kernel_bundle.h"
+#include "kernels.h"
+
+TEST_CASE(
+    "Check is_compatible(const std::vector<kernel_id>, const device) for zero "
+    "kernels",
+    "[kernel_bundle]") {
+  const sycl::device device = sycl_cts::util::get_cts_object::device();
+  const std::vector<sycl::kernel_id> kernelIds;
+
+  CHECK(sycl::is_compatible(kernelIds, device));
+}
+
+TEST_CASE(
+    "Check is_compatible(const std::vector<kernel_id>, const device) for "
+    "built-in kernel ids for the selected device",
+    "[kernel_bundle]") {
+  const sycl::device device = sycl_cts::util::get_cts_object::device();
+  const std::vector<sycl::kernel_id> builtinKernelIds =
+      device.get_info<sycl::info::device::built_in_kernel_ids>();
+
```

Comment:
Shouldn't we check that `builtinKernelIds` is not empty? 
Otherwise, this becomes a duplicate of the previous test.

Reply:
You are right, added check with SKIP.